pyanaconda/installation.py

```diff
@@ -234,22 +235,17 @@ def _prepare_installation(payload, ksdata):
     installation_queue.task_completed.connect(lambda x: progress_step(x.name))
 
     # This should be the only thread running, wait for the others to finish if not.
+    #
+    # We need to do this before we start filling the task queue, or else the DBus
+    # tasks we create might have stale data due to background processing threads
+    # still running at task creation time.
     if threadMgr.running > 1:
-        # it could be that the threads finish execution before the task is executed,
-        # but that should not cause any issues
-
-        def wait_for_all_treads():
-            for message in ("Thread %s is running" % n for n in threadMgr.names):
-                log.debug(message)
+        # show a progress message
+        progressQ.send_message(N_("Waiting for %s threads to finish") % (threadMgr.running - 1))
```

Comment:
I'm not sure if this is not a too technical message for a user. Maybe `Waiting for %s background jobs to finish` would be less technical? Not sure really.

Reply:
These are the same messages as before. Do we really want to change them in RHEL 8?

Reply:
Agreed with @poncovka, I think we want to keep this externally consistent, at least on the RHEL branches.

Reply:
OK, good point.